components/saveFile.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, Tooltip } from '@nextui-org/react';
+import React from 'react';
+import { GoDownload } from 'react-icons/go';
+
+interface Props {
```

Comment:
nit: `SaveFileProps` so we can export it if/when needed

Reply:
Done!